src/libstd/io/cursor.rs

```diff
@@ -34,16 +81,50 @@ pub struct Cursor<T> {
 
 impl<T> Cursor<T> {
     /// Creates a new cursor wrapping the provided underlying I/O object.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io::Cursor;
+    ///
+    /// let buff = Cursor::new(vec![]);
```

Comment:
s/vec![]/Vec::new()/ ?

I personally prefer `Vec::new()` but this changes from `vec![]` to `Vec::new()` below, so I figure it'd be good to at least be consistent (but I don't mind which way)